files/ids/bgee/ea.htm

```diff
@@ -88,10 +88,13 @@
 Hostile creatures, but uses blue (neutral) selection circles.
 </div>
 <br />
+<div class="idsHeader"><a name="254">254 CHARMED_PC</a></div>
+<div class="indent1">
+    This is just a separate EA from ENEMY for detection purposes. They're still valid objects for [EVILCUTOFF] and NearestEnemyOf(), but not by [ENEMY]. It's not specific to PC's.
+</div>
```

Comment:
I agree about the naming — we have the same in gemrb, but this is inconsistent with the stuff you copied earlier — it used EVILBYCHARM.

Reply:
> I agree about the naming — we have the same in gemrb, but this is inconsistent with the stuff you copied earlier — it used EVILBYCHARM.

I'll change EVILBYCHARM then...